include/box2d/types.h

```diff
@@ -1440,9 +1448,15 @@ typedef struct b2DebugDraw
 	/// Option to draw contact normal impulses
 	bool drawContactImpulses;
 
+	/// Option to draw contact feature ids
+	bool drawContactFeatures;
```

Comment:
Adding this before drawFrictionImpulses will subtly break bindings generated for 3.0.0 that point to a 3.1.0 shared library, which might be unexpected if people expect Open/Close Principle adherence.

Reply:
I cannot anticipate all the assumptions made by binding generators. I'll make a note of this in the 3.1 release notes.

Reply:
I had made my bindings against the partial 3.1.0 in main, which was an error on my part. I've changed everything so my Github actions build the latest tag instead, and my bindings work against 3.0.0, with a unit test to detect if b2Version returns something other than 3.0.0. When you're ready to release 3.1.0, I'll get a failing build and I'll update my code accordingly. The two I've mentioned in this PR would be the tip of the iceberg if we were to analyse the gap between 3.0.0 and 3.1.0, so if you're going to make a note in the release notes, it should probably either be vague about _a number of changes_ that will require recreation of any bindings, or I can help you an exhaustive list.

Reply:
I expect any point release to have breaking changes. I feel a physics engine is a bit different than something like SDL where API stability is at a premium. For Box2D features, performance, and robustness are at a premium and there are a lot of features to come.